integration_tests/pages/page.ts

```diff
@@ -1,19 +1,51 @@
+import 'cypress-axe'
+import { Result } from 'axe-core'
+
 export type PageElement = Cypress.Chainable<JQuery>
 
 export default abstract class Page {
   static verifyOnPage<T>(constructor: new () => T): T {
     return new constructor()
   }
 
-  constructor(private readonly title: string) {
+  constructor(
+    private readonly title: string,
+    private readonly skipA11y = false,
+  ) {
     this.checkOnPage()
   }
 
   checkOnPage(): void {
     cy.get('h1').contains(this.title)
+    if (!(Cypress.env('SKIP_AXE') || this.skipA11y)) {
```

Comment:
we've got a `SKIP_AXE` and a `skipA11y`, wondering if it is better to have the same name for both i.e. `skipAxe` or maybe even `SKIP_ACCESSIBILITY_CHECKS` and `skipAccessibilityChecks`